<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="renshaohai-PC" name="com.emar.test.testcases.mediapurchasingmanagement.DBtestDemo" tests="1" failures="1" timestamp="7 May 2015 10:16:14 GMT" time="0.000" errors="0">
  <testcase name="test_2_addSQE_SQL" time="0.000" classname="com.emar.test.testcases.mediapurchasingmanagement.DBtestDemo">
    <failure type="java.lang.RuntimeException" message="java.lang.NoSuchMethodError: org.databene.jdbacl.DBUtil.getConnectData(Ljava/lang/String;)Lorg/databene/commons/JDBCConnectData;">
      <![CDATA[java.lang.RuntimeException: java.lang.NoSuchMethodError: org.databene.jdbacl.DBUtil.getConnectData(Ljava/lang/String;)Lorg/databene/commons/JDBCConnectData;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.lang.NoSuchMethodError: org.databene.jdbacl.DBUtil.getConnectData(Ljava/lang/String;)Lorg/databene/commons/JDBCConnectData;
	at org.databene.platform.db.DBSystem.setEnvironment(DBSystem.java:192)
	at org.databene.platform.db.DBSystem.<init>(DBSystem.java:152)
	at org.databene.benerator.anno.AnnotationMapper.parseDatabase(AnnotationMapper.java:397)
	at org.databene.benerator.anno.AnnotationMapper.parseClassAnnotations(AnnotationMapper.java:136)
	at org.databene.feed4testng.FeedTest.feeder(FeedTest.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:153)
	... 20 more
]]>
    </failure>
  </testcase> <!-- test_2_addSQE_SQL -->
</testsuite> <!-- com.emar.test.testcases.mediapurchasingmanagement.DBtestDemo -->
